roxie/ccd/ccdstate.cpp

```diff
@@ -1866,6 +1866,8 @@ class CRoxiePackageSetWatcher : public CInterface
                             }
                             if (oldPackageManager && oldPackageManager->matches(xmlHash, isActive))
                             {
+                                //Check for any changes to the queryset
+                                oldPackageManager->reloadIncremental();
```

Comment:
@ghalliday since we are trying to keep even inactive packages up to date (I believe in case we have to quickly switch because the inactive package becomes active)... it doesn't look like the inactive package  will get the incremental changes.  If we later switch to it the new queries may not be there.

Reply:
Tested activating an inactive packagemap seemed to work.  I'll approve.